src/analysis/normed_space/linear_isometry.lean

```diff
@@ -310,12 +314,12 @@ e.isometry.diam_image s
 
 variables {α : Type*} [topological_space α]
 
-@[simp] lemma comp_continuous_on_iff {f : α → E} {s : set α} :
-  continuous_on (e ∘ f) s ↔ continuous_on f s :=
-e.isometry.comp_continuous_on_iff
+@[simp] lemma comp_continuous_on_iff {f₁ : α → E₁} {s : set α} :
+  continuous_on (e₁ ∘ f₁) s ↔ continuous_on f₁ s :=
+e₁.isometry.comp_continuous_on_iff
```

Comment:
Same here (and for `isometry.comp_continuous(_on)_iff`).

Reply:
I see you also did this, thank you!